source/kubernetes/index.html.md

```diff
@@ -1,31 +1,66 @@
 ---
-title: GOV.UK Kubernetes cluster documentation for developers
+title: GOV.UK Internal Developer Platform
 weight: 1
 layout: multipage_layout
 ---
 
-# GOV.UK Kubernetes cluster documentation for developers
+# GOV.UK Internal Developer Platform
 
-Use this documentation to find out how to:
+The GOV.UK Internal Developer Platform is a centralised platform where teams can build, deploy, and manage applications in a reliable and secure environment.
 
-- start using the GOV.UK Kubernetes clusters
-- manage your GOV.UK app on Kubernetes
+This documentation is designed for both platform users and internal team members, providing guides, tools, and best practices to onboard, manage, and troubleshoot your applications effectively.
 
-## Introduction
```

Comment:
Is there a reason for removing the intro section about Kubernetes? I think it was be quite informative, especially for less experienced devs. We have apprentices and juniors who would benefit from reading up on the technology before jumping into the user guide.

Reply:
Indeed, it's helpful information, but it's not that helpful on the index page. Kubernetes scares people off at the best times, but throwing it into an introduction is unnecessary and distracts people away from what's important. I prefer to explain the abstraction (platform level) here, the what and why, and leave the technology choices to those who want to know about them. Technology choices (in my opinion) are far better placed in the "How the platform works" section.

Reply:
Perhaps we could have a section in the "How the platform works" section explaining our technology decision-making.